dotCMS/src/main/java/com/dotcms/security/apps/AppsKeyDefaultProvider.java

```diff
@@ -0,0 +1,27 @@
+package com.dotcms.security.apps;
+
+import com.dotcms.auth.providers.jwt.factories.SigningKeyFactory;
+import com.dotmarketing.business.APILocator;
+import java.security.Key;
+
+public class AppsKeyDefaultProvider implements SigningKeyFactory {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A class which only has private constructors should be final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5677177)